src/components/AppointmentConfigModal.vue

```diff
@@ -214,8 +214,12 @@ export default {
 			}
 
 			const uri = this.editing.targetCalendarUri
-			const calendar = this.ownSortedCalendars.find(cal => this.calendarUrlToUri(cal.url) === uri)
-			return calendar || this.ownSortedCalendars[0]
+			let calendar = this.ownSortedCalendars.find(cal => this.calendarUrlToUri(cal.url) === uri)
+			if (!calendar) {
+				calendar = this.ownSortedCalendars[0]
```

Comment:
`calender` could be undefined if the user has no more calendars

Reply:
I just tried to reproduce this but the whole appointment section vanishes from the left sidebar when the last calendar is deleted. So this shouldn't be an issue because the modal can't be opened if no calendar is left.

https://github.com/nextcloud/calendar/blob/7008557529bc57aaeb69a30f77bb4943f06d21fa/src/components/AppNavigation/AppointmentConfigList.vue#L28

Reply:
Reloading the page will create the default (personal) calendar again and the appointment list can be seen and edited again.